Update bd.m3u

badhon495 opened the pull request:
i updated this with all channels i have.some of them only works with bd ip

Commit:
Update bd.m3u

freearhey commented:
Unfortunately, I cannot approve this pull request.

First of all, we do not add geo-blocked channels to playlists so it makes it very difficult to support them further.

Secondly, the channels in playlists are sorted by country of broadcast source and not by country of broadcast available for viewing. In your list most of the channels are not from Bangladesh.

To avoid these mistakes in the future, please read this guide before making any changes: https://github.com/iptv-org/iptv/blob/master/.github/CONTRIBUTING.md#add-channel

badhon495 commented:
brother i updated again.
now i removed non Bengali channel.
and they are not area restricted now.
thanks :)

On Tue, Jun 16, 2020, 6:32 PM Aleksandr Statciuk <notifications@github.com>
wrote:

> Closed #1522 <https://github.com/iptv-org/iptv/pull/1522>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/iptv-org/iptv/pull/1522#event-3448746051>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AP7FZFAQJRUINVKKUCMXPRLRW5Q5HANCNFSM4N7QDK2A>
> .
>
